Do not depend on hspec-discover (the haskell library), as that is empty
anyways. We ensure that the hspec-discover binary is available via
control/debian.

Index: hspec-2.1.7/hspec.cabal
===================================================================
--- hspec-2.1.7.orig/hspec.cabal	2015-06-29 13:58:58.679142882 +0200
+++ hspec-2.1.7/hspec.cabal	2015-06-29 13:59:03.775258382 +0200
@@ -42,7 +42,6 @@
   build-depends:
       base == 4.*
     , hspec-core == 2.1.7
-    , hspec-discover == 2.1.7
     , hspec-expectations == 0.6.1.*
     , transformers >= 0.2.2.0
     , QuickCheck >= 2.5.1
